Index: src/main/resources/universidad.sql
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/universidad.sql b/src/main/resources/universidad.sql
new file mode 100644
--- /dev/null	(date 1695039445655)
+++ b/src/main/resources/universidad.sql	(date 1695039445655)
@@ -0,0 +1,169 @@
+-- phpMyAdmin SQL Dump
+-- version 5.2.1
+-- https://www.phpmyadmin.net/
+--
+-- Servidor: 127.0.0.1
+-- Tiempo de generación: 18-09-2023 a las 14:17:25
+-- Versión del servidor: 10.4.28-MariaDB
+-- Versión de PHP: 8.2.4
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Base de datos: `universidad`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Estructura de tabla para la tabla `grades`
+--
+
+CREATE TABLE `grades` (
+  `id_grades` int(11) NOT NULL,
+  `id_student` int(11) NOT NULL,
+  `id_subject` int(11) NOT NULL,
+  `corte` varchar(25) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci;
+
+--
+-- Volcado de datos para la tabla `grades`
+--
+
+INSERT INTO `grades` (`id_grades`, `id_student`, `id_subject`, `corte`) VALUES
+(1, 1, 1, '5'),
+(2, 2, 2, '5');
+
+-- --------------------------------------------------------
+
+--
+-- Estructura de tabla para la tabla `student`
+--
+
+CREATE TABLE `student` (
+  `id_student` int(11) NOT NULL,
+  `name` varchar(50) NOT NULL,
+  `email` varchar(80) NOT NULL,
+  `semester` int(2) NOT NULL,
+  `career` varchar(25) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci;
+
+--
+-- Volcado de datos para la tabla `student`
+--
+
+INSERT INTO `student` (`id_student`, `name`, `email`, `semester`, `career`) VALUES
+(1, 'Juan Felpe', 'jrojas@cue.edu.co', 1, 'Ing. Software'),
+(2, 'Ricardo ', 'Rorozco@cue.edu.co', 1, 'Ing. Software'),
+(3, 'Juan', 'Jgallego@cue.edu.co', 1, 'Administración'),
+(4, 'Samuel', 'Scaicedo@cue.edu.co', 1, 'Veterinaria'),
+(5, 'Daniel', 'DBueno@cue.edu.co', 1, 'Medicina');
+
+-- --------------------------------------------------------
+
+--
+-- Estructura de tabla para la tabla `subject`
+--
+
+CREATE TABLE `subject` (
+  `id_subject` int(11) NOT NULL,
+  `name` varchar(50) NOT NULL,
+  `id_teacher` varchar(25) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci;
+
+--
+-- Volcado de datos para la tabla `subject`
+--
+
+INSERT INTO `subject` (`id_subject`, `name`, `id_teacher`) VALUES
+(1, 'Bases de datos', '1'),
+(2, 'Programación', '2');
+
+-- --------------------------------------------------------
+
+--
+-- Estructura de tabla para la tabla `teachers`
+--
+
+CREATE TABLE `teachers` (
+  `id_teacher` int(11) NOT NULL,
+  `name` varchar(80) NOT NULL,
+  `email` varchar(50) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci;
+
+--
+-- Volcado de datos para la tabla `teachers`
+--
+
+INSERT INTO `teachers` (`id_teacher`, `name`, `email`) VALUES
+(1, 'Monica Tobon', 'mtobon@cue.edu.co'),
+(2, 'Arle Morales', 'amorales@cue.edu.co');
+
+--
+-- Índices para tablas volcadas
+--
+
+--
+-- Indices de la tabla `grades`
+--
+ALTER TABLE `grades`
+  ADD PRIMARY KEY (`id_grades`);
+
+--
+-- Indices de la tabla `student`
+--
+ALTER TABLE `student`
+  ADD PRIMARY KEY (`id_student`);
+
+--
+-- Indices de la tabla `subject`
+--
+ALTER TABLE `subject`
+  ADD PRIMARY KEY (`id_subject`);
+
+--
+-- Indices de la tabla `teachers`
+--
+ALTER TABLE `teachers`
+  ADD PRIMARY KEY (`id_teacher`);
+
+--
+-- AUTO_INCREMENT de las tablas volcadas
+--
+
+--
+-- AUTO_INCREMENT de la tabla `grades`
+--
+ALTER TABLE `grades`
+  MODIFY `id_grades` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=3;
+
+--
+-- AUTO_INCREMENT de la tabla `student`
+--
+ALTER TABLE `student`
+  MODIFY `id_student` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=6;
+
+--
+-- AUTO_INCREMENT de la tabla `subject`
+--
+ALTER TABLE `subject`
+  MODIFY `id_subject` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=3;
+
+--
+-- AUTO_INCREMENT de la tabla `teachers`
+--
+ALTER TABLE `teachers`
+  MODIFY `id_teacher` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=3;
+COMMIT;
+
+/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
+/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
+/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
